terraform-infra-approvals/hmcts/tax-tribunals-shared-infrastructure.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources": [
+    {"type": "action-group-kv"},
```

Comment:
that doesn't look correct to terraform resource types?

Reply:
Hi Thomas,

I'm trying to resolve the error in this build:
https://build.hmcts.net/job/HMCTS_j_to_z/job/tax-tribunals-shared-infrastructure/job/PR-19/2/pipeline-console/?selected-node=10

```
this repo is using a terraform resource that is not allowed,
[3](https://build.hmcts.net/job/HMCTS_j_to_z/job/tax-tribunals-shared-infrastructure/job/PR-19/2/pipeline-console/?start-byte=0&selected-node=88#log-3)
whitelists are stored in https://github.com/hmcts/cnp-jenkins-config/tree/master/terraform-infra-approvals
[4](https://build.hmcts.net/job/HMCTS_j_to_z/job/tax-tribunals-shared-infrastructure/job/PR-19/2/pipeline-console/?start-byte=0&selected-node=88#log-4)
send a pull request if you think this is in error.
[5](https://build.hmcts.net/job/HMCTS_j_to_z/job/tax-tribunals-shared-infrastructure/job/PR-19/2/pipeline-console/?start-byte=0&selected-node=88#log-5)
non whitelisted resources:
[6](https://build.hmcts.net/job/HMCTS_j_to_z/job/tax-tribunals-shared-infrastructure/job/PR-19/2/pipeline-console/?start-byte=0&selected-node=88#log-6)

[{managed azurerm_resource_group_template_deployment action-group-kv {azurerm } {groups.tf 16}} {managed azurerm_resource_group_template_deployment action-group-storage {azurerm } {groups.tf 31}} {managed azurerm_resource_group_template_deployment action-group-db {azurerm } {groups.tf 1}}]
```